src/ViewTrait.php

```diff
@@ -426,7 +443,7 @@ public function renderFile(string $viewFile, array $parameters = []): string
         }
 
         if (is_file($viewFile)) {
-            $viewFile = $this->localize($viewFile);
+            $viewFile = $this->localize($viewFile, $this->localeState->getLanguage());
```

Comment:
Passing from locale state is redundant. Already used as a fallback language in this method.